tests/conftest.py

```diff
@@ -58,11 +58,16 @@ def cleanup():
         Make sure to delete the created account abd buckets
         """
         try:
+            # Delete all the accounts that were created by this fixture
             bucket_manager = BucketManager()
-            for bucket in bucket_manager.list():
-                bucket_manager.delete(bucket, force=True)
-            for acc in acc_manager_instance.list():
+            for acc in acc_manager_instance.accounts_created:
+                # Delete all the buckets that were created by this account
+                acc_id = acc_manager_instance.status(account_name=acc)["_id"]
+                for bucket in bucket_manager.list(use_wide=True):
+                    if bucket["owner_account"] == acc_id:
+                        bucket_manager.delete(bucket["name"], force=True)
```

Comment:
If there is a risk that during iteration one item will not get deleted then I think it's better to cover inner block with try except like this:
```suggestion
                        try:
                            bucket_manager.delete(bucket["name"], force=True)
                        except Exception as ex:
                            log.exception("Failed to delete bucket")
```

Advantage of it is that if you have e.g. 10 buckets and it will fail in first one - it will not continue to delete the rest of 9 buckets.

But if you have try/except within the for loop, it will continue to iterate and try to delete next 9 buckets as well.

Reply:
Thanks for pointing it out - added in https://github.com/red-hat-storage/noobaa-sa-ci/pull/49/commits/3b60a4ed4f03f61edca0665cdb7bae7f6925cd44